changelog.json

```
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "2.9.13-beta2": [
+      "[Added] Add include and exclude to the context menu of multiple selected files in the changes list - #7790. Thanks @tsvetilian-ty!",
```

I wanted to make this a bit more utility-focused, but we don't have to use this.
```suggestion
      "[Added] Add ability to include or exclude multiple selected files for a commit from the context menu - #7790. Thanks @tsvetilian-ty!",
```